app/src/main/res/layout/fragment_downloads.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/theme_background">
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="56dp"
+        android:background="?colorPrimary"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler_view"
+        android:layout_width="match_parent"
```

Comment:
It's usually recommended to use 0dp instead of `match_parent` when using a ConstraintLayout. 
Using `match_parent` sometimes leads to weird sculling issues when a RecyclerView is inside a ConstraintLayout

Reply:
sure

Reply:
When I set width to 0dp, I get a MissingConstraints warning. In the settings fragment it is also handled the same.

Reply:
Are you adding the following as well?

```xml
app:layout_constraintStart_toStartOf="parent"
app:layout_constraintEnd_toEndOf="parent"
```